src/app/responders/common/dndefine.py

```diff
@@ -111,4 +111,17 @@ class DndefineSlackResponder(DndefineResponder, SlackResponder):
     'archery': 'Ranger: You gain a +2 bonus to attack rolls you make with ranged weapons.',
     'cure_wounds': '1st level evocation\nCasting Time: 1 action\nRange: touch\nComponents: V,S\nDuration: Instantaneous\nDescription: A creature you touch regains a number of hit points equal to 1d8 + your spellcasting ability modifier. This spell has no effect on undead or constructs.',
     'hunters_mark': '1st level divination\nCasting Time: 1 bonus Action\nRange: 90 feet\nComponents: V\nDuration: Concentration, up to 1 hour\nDescription: You choose a creature you can see within range and mystically mark it as your quarry. Until the spell ends, you deal an extra 1d6 damage to the target whenever you hit it with a weapon attack, and you have advantage on any Wisdom (Perception) or Wisdom (Survival) check you make to find it. If the target drops to 0 hit points before this spell ends, you can use a bonus action on a subsequent turn of yours to mark a new creature.',
+    'pact_of_the_tomb': 'Your patron gives you a grim oire called a B ook of Shadows. W hen you gain this feature, choose three cantrips from any class\'s spell list. W hile the book is on your person, you can cast those cantrips at will. They don\'t count against your number of cantrips known.\nIf you lose your B ook o f Shadows, you can perform a 1-hour cerem ony to receive a replacement from your patron. This cerem ony can be perform ed during a short or long rest, and it destroys the previous book. The book turns to ash when you die.',
```

Comment:
There's some weird spacing on this one. `grim oire`, `B ook`, etc. If you care to fix!